### DISCUSSION TITLE
need a little help with converting to sRBG

### DISCUSSION THREAD

**Discussion by toy-opensource** [SixLabors/ImageSharp#1585] [Q&A] [answered]
hey you guys
lemme just first say that im new with colorspaces so bear wtih me

im getting input images from a lot of random places
(and yes they might be pngs or jpgs or gifs)
im basically trying to normalize them - 
so getting them all to be sRGB jpgs of a certain size

now getting them to be jpegs and of a certain size with image sharp has been no probelm
but getting them to sRGB has been a lot more difficult

like for sizing i just did an img.Mutate() and that worked
but there is nothing like img.ToSRGB()
there is [srgbworkingspace](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.ColorSpaces.Conversion.SRgbWorkingSpace.html) and [companding](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.ColorSpaces.Companding.html)
but im confused as to what to put in for parameters

can someone help me out and explain some of this to me 
or point some example i can see whats up?

**Comment by JimBobSquarePants** [comment, marked as answer]
Hi @toy-opensource 

There's actually an open WIP (_work in progress_) PR https://github.com/SixLabors/ImageSharp/pull/1567 which would allow us to provide this functionality automatically upon decode but it remains unfinished with no clear plan to deliver it.

Theoretically you could use the existing colorspace transform methods to transform. 

I'm no expert on the subject but you should be able get some of the data you require from the `Image.Metdata.IccProfile.Header` property.

**Reply by toy-opensource** [reply]
great jim! 
thx for the direction - ill give it a shot
